### DISCUSSION TITLE
Workflow does not get triggered, even though the configuration is correct & it worked before.

### DISCUSSION THREAD

**Discussion by jongdetim** [community/community#40292] [Actions]
### Select Topic Area

Bug

### Body

I renamed a repository. This broke workflows and pages. I endlessly tried reconfiguring the static.yml workflow file, but it simply refuses to be triggered unless manually. See https://github.com/jongdetim/mini-keras/blob/main/.github/workflows/static.yml for the workflow file. When something gets pushed to the ./docs/ directory, it's supposed to redeploy the documentation as a github page, to reflect the changes. This doesn't happen anymore after changing the repository name.

**Comment by ossanna16** [comment]
Hi there @jongdetim and welcome to our community! Thank you for reporting this bug!
